lib/services/presets.go

```diff
@@ -230,6 +296,24 @@ func NewPresetGroupAccessRole() types.Role {
 	return role
 }
 
+// defaultRoleMetadataLabels are metadata labels that should be applied to each
+// role by default.
+func defaultRoleMetadataLabels() map[string]map[string]string {
```

Comment:
this is semantically different than the other `defaultXXX` functions, maybe it needs another name like `bootstrapRoleLabels`